spec/requests/users_controller_spec.rb

```diff
@@ -5061,6 +5061,39 @@ def revoke(user, skip_remote: false)
       expect(response.parsed_body['user_bookmark_list']['bookmarks'].map { |b| b['id'] }).to match_array([bookmark1.id, bookmark2.id])
     end
 
+    it "returns an .ics file of bookmark reminders for the user in date order" do
+      bookmark1.update(name: nil, reminder_at: 1.day.from_now)
+      bookmark2.update(name: "Some bookmark note", reminder_at: 1.week.from_now)
```

Comment:
```suggestion
      bookmark1.update!(name: nil, reminder_at: 1.day.from_now)
      bookmark2.update!(name: "Some bookmark note", reminder_at: 1.week.from_now)
```

Reply:
Nooo I thought I caught all of these, I knew you would be on the lookout XD